### ISSUE TITLE
Moderate oligozoospermia

### ISSUE THREAD

**Issue by MagoWyr** [obophenotype/human-phenotype-ontology#9799]
**Preferred term label:**
Moderate oligozoospermia

**Synonyms**

**Definition (free text, please give PubMed ID)**
>10 - <29 million sperm (>5 - <10 million/mL concentration)

**Parent term (use hpo.jax.org/app)**
HP:0000798

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA: 399805

**Your nano-attribution (ORCID)**
ORPHA: 399805

**Comment by pnrobinson** [comment]
@MagoWyr 
added with definition
A type of oligozoospermia characterized by a total number of more than 10 but less than 29 million sperm in the ejaculate or a concentration of more than 5 but less than 10 million sperm per milliliter.
